x-pack/plugin/eql/qa/common/src/main/java/org/elasticsearch/test/eql/CommonEqlActionTestCase.java

```diff
@@ -162,8 +152,9 @@ public void cleanup() throws Exception {
                 name = spec.note();
             }
             if (Strings.isNullOrEmpty(name)) {
-                name = spec.query();
+                name = "" + (counter.get() + 1);
```

Comment:
Changed this one since Eclipse cannot run it (it's an invalid method name according to the Java conventions).